File: helfomat-web/src/main/java/de/helfenkannjeder/helfomat/HelfomatApplication.java
package de.helfenkannjeder.helfomat;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.kafka.KafkaAutoConfiguration;

/**
 * @author Valentin Zickner
 */
@SpringBootApplication(exclude = KafkaAutoConfiguration.class)
public class HelfomatApplication {

    public static void main(String[] args) {
        SpringApplication.run(HelfomatApplication.class, args);
    }

}

File: helfomat-web/src/main/kotlin/de/helfenkannjeder/helfomat/HelfomatApplication.kt
package de.helfenkannjeder.helfomat

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

/**
 * @author Valentin Zickner
 */
@SpringBootApplication
open class HelfomatApplication

fun main(args: Array<String>) {
    runApplication<HelfomatApplication>(*args)
}